test/specs/controller.financial.tests.js

```diff
@@ -1,28 +1,22 @@
 describe('Financial controller tests', function() {
 	it('Should create candlestick elements for each data item during initialization', function() {
+		var date = new Date();
 		var chart = window.acquireChart({type: 'candlestick',
 			data: {
 				datasets: [{
 					data: [
-						{t: 1491004800, o: 70.87, h: 80.13, l: 40.94, c: 50.24},
-						{t: 1491091200, o: 60.23, h: 200.23, l: 50.29, c: 90.20},
-						{t: 1491177600, o: 40.00, h: 50.42, l: 20.31, c: 30.20},
-						{t: 1491264000, o: 60.90, h: 80.34, l: 40.14, c: 60.29},
-						{t: 1491350400, o: 60.94, h: 100.09, l: 50.78, c: 90.04},
-						{t: 1491436800, o: 30.49, h: 50.20, l: 20.09, c: 40.20},
-						{t: 1491523200, o: 50.04, h: 80.93, l: 40.94, c: 70.24},
-						{t: 1491609600, o: 60.29, h: 100.49, l: 50.49, c: 90.24},
-						{t: 1491696000, o: 40.04, h: 50.23, l: 20.23, c: 30.49},
-						{t: 1491782400, o: 30.23, h: 50.09, l: 20.87, c: 40.49}
+						{x: date, t: 1491004800, o: 70.87, h: 80.13, l: 40.94, c: 50.24},
```

Comment:
Hmm. We don't want to have to specify `x` and `t`. And `x` shouldn't be the same `date` everywhere. 

Ideally we could change the scale ID to `t` and have that work still. We may have to set the `position` to `'bottom'` or `axis` to `'x'`: https://www.chartjs.org/docs/latest/axes/cartesian/.  Though I'm not quite sure that'd be enough. I think the bar chart may look for the parameter to be called `x`. I just changed the samples to `x` in https://github.com/chartjs/chartjs-chart-financial/commit/0b4555efbfb82a7b665ec4bda475c17e17267553 but am not sure I'm really happy with that.

Or we could turn `parsing` on to make `t` work: https://www.chartjs.org/docs/latest/general/data-structures.html. But that would have some performance impact

I haven't really thought through what the best solution is here

Reply:
Took the most naive approach and matched the sample for the test as well, using x for the time axis. Checking other plugins and the default examples, it seemed to make the most sense for consistency.